tests/python/gaia-ui-tests/gaiatest/apps/email/regions/setup.py

```diff
@@ -182,7 +183,10 @@ def tap_next(self):
                 *self._account_prefs_next_locator))))
 
     def check_for_emails_interval(self, value):
-        self.marionette.execute_script('document.querySelector("[data-l10n-id = settings-check-every-5min]").value = "%s";' % value)
+        self.marionette.execute_script("""
+            SpecialPowers.setIntPref('dom.requestSync.minInterval', 1);
+            document.querySelector("[data-l10n-id = settings-check-every-5min]").value = 20000;
```

Comment:
The pref is for the minimum interval time for syncing, which is normally 100s. We need to set it lower than 20s, to be able to sync to 20s.